edina/src/data/Su/json/collect.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "name":"唐韵",

```

Comment:
Where is the image?

Reply:
no img
https://suxing.artron.net/collect